app/models/Campaign.js

```diff
@@ -34,8 +34,12 @@ const campaignSchema = new mongoose.Schema({
   msg_menu_signedup_external: String,
   msg_menu_signedup_gambit: String,
   msg_no_photo_sent: String,
-  msg_relative_reminder_signup: String,
-  msg_relative_reminder_reportback: String,
+
+  // Exposed messages.
+  messages: {
```

Comment:
Is this an actual "reminder" to signup (suggesting user has already gotten a message to signup)?

Or just the first broadcast message sent prompting a user to signup for a new campaign? (Where using the term "reminder" is misleading)